docs/the-new-architecture/pillars-turbomodule.md

```diff
@@ -223,7 +260,7 @@ android
         ├── AndroidManifest.xml
         └── java
             └── com
-                └── rtncalculator
+                └── RTNCalculator
```

Comment:
Commenting here but valid for the rest of PR:
I'm against those changes specifically as:
- packages in java/kotlin are all lowercase
- Class prefixes in Java/Kotlin like `RTN` are uncommon. It's a pattern in iOS but not on Android. I'm not sure we want Android to follow the iOS convention. We want instead to follow the platform specific conventions.

Reply:
@cortinico I understand, the problem itself was maintaining the standard of declarations such as class names. RTNCalculatorPackage in one location and CalculatorPackage in another.

Reply:
I understand, but the React Native framework is following the same convention. If you use any scaffodling tool, you would follow the same convention.
The rest of the website, also follows this convention. I'd refrain from breaking it here

Reply:
Done @cortinico, thank you!